lib/web_ui/lib/src/engine/platform_views/content_manager.dart

```diff
@@ -65,6 +65,11 @@ class PlatformViewManager {
     return _contents.containsKey(viewId);
   }
 
+  /// Returns the pre-rendered contents of [viewId], to inject them into the DOM.
+  DomElement getContents(int viewId) {
```

Comment:
For consistency with the method below, should we add an assert-with-a-message here?